src/Components/JobsPage/JobsPage.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
please create a .local stories file so that this can work wiht real APIs when we get it running.